README.md

```diff
@@ -115,56 +121,67 @@ The configuration file is a simple json document, a sample can be seen bellow:
-    * secure [not supported yet] [default: true] - enable or disable https
-    * timeout [not supported yet] [default: 2000ms] - timeout for http transport
-    * token - An authentication token to send to Statful
-    * app [optional] - if specified set a tag ‘app=foo’
-    * dryrun [optional] [default: false] - debug log metrics when flushing the buffer
-    * tags [optional] - global list of tags to set, these are merged with custom tags set on method calls with priority to custom tags
-    * sampleRate [optional] [default: 100] [between: 1-100] - global rate sampling
-    * namespace [optional] [default: 'application'] - default namespace
-    * flushSize [optional] [default: 10] - defines the periodicity of buffer flushes by buffer size
-    * flushInterval [optional] [default: 0] - defines an interval to periodically flush the buffer based on time
-    * maxBufferSize [optional] [default: 5000] - defines how many metrics at max are kept in the buffer between forced flushes
-
-Vertx Statful specific configurations:
-
-    * enabled [optional] [default: true] - enable/disable the client
-    * timerAggregations [optional] [default: [Aggregation.AVG, Aggregation.P90, Aggregation.COUNT] ] - aggregations to be applied to timer based metrics
-    * timerFrequency [optional] [default: AggregationFreq.FREQ_10] - aggregation frequency for timer based metrics
-    * counterAggregations [optional] [default: [Aggregation.COUNT, Aggregation.SUM] ] - aggregations to be applied to counter based metrics
-    * counterFrequency [optional] [default: AggregationFreq.FREQ_10] - aggregation frequency for counter based metrics
-    * gaugeAggregations [optional] [default: [Aggregation.LAST, Aggregation.MAX, Aggregation.AVG] ] - aggregations to be applied to gauge based metrics
-    * gaugeFrequency [optional] [default: AggregationFreq.FREQ_10] - aggregation frequency for gauge based metrics
-    * http-server-url-patterns [optional] - patterns to transform urls for metrics collection
-    * http-server-ignore-url-patterns [optional] - patterns of urls that you won't want tracked
-    * gauge-collection-interval [optional] [default: ] - to avoid reporting gauges every time it changes
-    * httpMetricsPath [optional] [default: '/tel/v2.0/metrics'] - path to send metrics to when http transport is set
-    * transport [optional] [default: HTTP] - type of transport to be used when sending metrics to statful (UDP/HTTP)
+## Configurations
+The following section presents detailed information on the available options for setup in the configuration parameters.
+
+### Global Configurations
+
+| Option | Description | Type | Default | Required |
+|:---|:---|:---|:---|:---|
+| _host_ | Defines the hostname to where the metrics are sent. | `string` | `api.statful.com` | **NO** |
+| _port_ | Defines the port to where the metrics are sent. | `string` | `443` | **NO** |
+| _secure_ | Enables or disables https. <br><br>Not yet supported. | `boolean` | `true` | **NO** |
```

Comment:
it isn't?

Reply:
I don't know, is it? It said so before xD 

>     * secure [not supported yet] [default: true] - enable or disable https